### PR TITLE
Cynerio Event Solution

### PR DESCRIPTION
**Pull request by meravcy**
Required items, please complete
   
   Change(s):
   - Cynerio solution

   Reason for Change(s):
   - New Sentinel solution, including data connector and ASIM parser

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - No

### CODE REVIEW

**Comment by meravcy** [comment]
> 

@microsoft-github-policy-service agree company="Cynerio"

**Comment by v-prasadboke** [comment]
Hello @meravcy as it as a new solution please add the required files of the solution ' main template, create ui, input file' .
Please go through this ' readme ' file to create a package for the solution
https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/README.md

**Comment by v-prasadboke** [comment]
image
Hello @meravcy please create a folder named Data as shown in the above image and move input file in that folder from the ' v2 ' folder  
And also move parser folder to this solution folder and 
Move sample data folder content to this path ' https://github.com/Azure/Azure-Sentinel/tree/master/Sample%20Data ' 
Thank you

**Comment by v-prasadboke** [comment]
Hello @oshezaf waiting for your approval

**Comment by v-prasadboke** [comment]
Hello @meravcy Please convert parser file from .yaml to .txt 
And also add path for parser in input file

**Comment by v-prasadboke** [comment]
And please repackage the solution again

**Comment by meravcy** [comment]
@v-prasadboke already done

**Comment by v-prasadboke** [comment]
Sorry @meravcy but there are still some changes pending

**Comment by v-prasadboke** [comment]
After making these changes requested above, Please repackage the solution again

**Comment by v-prasadboke** [comment]
Hello @meravcy The solution has not been repackaged, as parser content is missing from the maintemplate.
Please repackage the solution again using the V2 tool.

**Comment by meravcy** [comment]
hello @v-prasadboke, I deleted the old package and rerun the script (v2). The result doesn't change. This script doesn't include the parsers in the maintemplate file, it only adds it in the createUIDefinition file

**Comment by v-prasadboke** [comment]
Checking on it

File: Solutions/Cynerio/Data/Solution_Cynerio.json
Comment (v-prasadboke) [inline Solutions/Cynerio/Data/Solution_Cynerio.json:11]: The file path for parser is incorrect,
It should be ( Solutions ) rather than "Solution".
Please repackage the solution the script will work after this change.
Thank you.

Comment (meravcy) [reply Solutions/Cynerio/Data/Solution_Cynerio.json]: done, thank you

**Comment by meravcy** [comment]
hello @v-prasadboke ,I pushed the fix yesterday, is there anything else to be done from our side?

**Comment by v-prasadboke** [comment]
> hello @v-prasadboke ,I pushed the fix yesterday, is there anything else to be done from our side?

Hello @meravcy im still facing some issue while deploying the solution,
So still working on it

File: Solutions/Cynerio/SolutionMetadata.json
Comment (v-prasadboke) [inline Solutions/Cynerio/SolutionMetadata.json:14]: Please change the tier cynerio to Partner 
and repackage the solution again

**Comment by meravcy** [comment]
hello @v-prasadboke, I pushed the fix for you comment last week, anything else required from our side?

**Comment by v-prasadboke** [comment]
Ok @meravcy going with this

**Comment by v-prasadboke** [comment]
Hello @meravcy everything looks good 
Just unable to ingest sample data so can you share the query result of both the parser so that i can approve the pr

**Comment by meravcy** [comment]
hello @v-prasadboke , add parsers result on Sample Data folder: Cynerio_Authentication_query_data.csv & Cynerio_NetworkSession_query_data.csv

**Review by v-prasadboke — APPROVED** [review]

**Comment by v-prasadboke** [comment]
All looks good

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**
